hbase-server/src/main/java/org/apache/hadoop/hbase/master/HMaster.java

```diff
@@ -659,6 +660,7 @@ protected RSRpcServices createRpcServices() throws IOException {
   @Override
   protected void configureInfoServer() {
     infoServer.addUnprivilegedServlet("master-status", "/master-status", MasterStatusServlet.class);
+    infoServer.addPrivilegedServlet("rits", "/rits", RitServlet.class);
```

Comment:
Nit: Let's call it "/rit". 
Not a big deal, but other paths are not in the plural form.